funcx_endpoint/tests/funcx_endpoint/endpoint/test_interchange.py

```diff
@@ -0,0 +1,63 @@
+from funcx_endpoint.endpoint.endpoint_manager import EndpointManager
+from funcx_endpoint.endpoint.interchange import EndpointInterchange
+from importlib.machinery import SourceFileLoader
+import os
+import logging
+import sys
+import shutil
+import pytest
+import json
+from pytest import fixture
+from unittest.mock import ANY
+
+logger = logging.getLogger('mock_funcx')
+
+
+class TestStart:
+
+    @pytest.fixture(autouse=True)
+    def test_setup_teardown(self):
+        # Code that will run before your test, for example:
+
+        funcx_dir = f'{os.getcwd()}'
+        config_dir = os.path.join(funcx_dir, "mock_endpoint")
+        assert not os.path.exists(config_dir)
```

Comment:
This might be problematic if the user, ctrl+c'ed the test while it was running and teardown didn't get to teardown. Just something to add to the notes at least.